test/public/defaults.js

```diff
@@ -371,8 +370,13 @@ module.exports.getInnerText = getInnerText;
  * @return {Promise<void>} resolves once the text has been checked
  */
 module.exports.expectInnerText = async (page, selector, innerText) => {
-    const actualInnerText = await getInnerText(await page.waitForSelector(selector, { timeout: 200 }));
-    expect(actualInnerText).to.equal(innerText);
+    await page.waitForSelector(selector);
+    await page.waitForFunction(
+        (selector, innerText) => document.querySelector(selector).innerText === innerText,
+        {},
+        selector,
+        innerText,
+    );
```

Comment:
I am not sure about this function, I think it's good to have information what  is the actual innerText in case of failure. When using waitForFunction it can be done by catching timeOutError and checking actual innerText

Reply:
Done